pkg/controller/volume/persistentvolume/pv_controller_base.go

```diff
@@ -574,7 +575,10 @@ func (ctrl *PersistentVolumeController) claimWorker(ctx context.Context) {
 		if err == nil {
 			// The claim still exists in informer cache, the event must have
 			// been add/update/sync
-			ctrl.updateClaim(ctx, claim)
+			if err := ctrl.updateClaim(ctx, claim); err != nil || claim.Status.Phase == v1.ClaimPending {
+				logger.V(2).Info("Error update claim status", "claimKey", key, "err", err)
+				ctrl.claimQueue.AddAfter(claimToClaimKey(claim), time.Second*1)
```

Comment:
why not rely on exponential backoff of the queue? I.e. `ctrl.claimQueue.AddRateLimited(key)`

Reply:
@sttts  thx, updated